src/components/explorer/searchResultsTables/ExperimentsTable.js

```diff
@@ -73,14 +73,27 @@ const ExperimentsTable = ({ data, datasetID }) => {
         (state) => state.explorer.tableSortOrderByDatasetID[datasetID]?.["experiments"],
     );
 
+    const experimentTypeFilters = useDynamicFilterOptions(data, "experimentType");
+
+    const filterColumns = SEARCH_RESULT_COLUMNS_EXP.map((col) => {
+        if (col.dataIndex === "experimentType") {
```

Comment:
this is a confusing way to alter `SEARCH_RESULT_COLUMNS_EXP`. It'd be better to move the columns definition into a new memoized variable which has dependencies on `experimentTypeFilters`.